cli/providers/aws/translate.go

```diff
@@ -166,7 +175,7 @@ func loadIAM(t *gocf.Template, s *service.Service, e *environment.Environment) {
 			"Statement": map[string]interface{}{
 				"Effect":   "Allow",
 				"Action":   []string{"logs:CreateLogStream", "logs:PutLogEvents"},
-				"Resource": fmt.Sprintf("arn:aws:logs:us-west-2:*:log-group:i2kit-%s:log-stream:%s-*", s.Name, s.Name),
+				"Resource": fmt.Sprintf("arn:aws:logs:us-west-2:*:log-group:i2kit-%s:log-stream:i-*", s.Name),
```

Comment:
Do we need to parametrize the region here?

Reply:
yes, I will do that in a separate PR